### PR TITLE
Update governance page after AGM results

### PR DESCRIPTION
**Pull request by haubourg**
community vote according to https://blog.qgis.org/2022/10/04/qgis-annual-general-meeting-2022/

### CODE REVIEW

File: source/site/getinvolved/governance/governance.rst [hunk 1]
```diff
@@ -26,12 +26,12 @@ To contact the board, please write to board@qgis.org.
...
 --------------------------
 
-* Marco Bernasocchi (Chair) [Chair since 5.2020; Vice chair 4.2018 - 4.2020]
-* Alessandro Pasotti (Vice chair) [Vice chair since 5.2020]
-* Andreas Neumann (Treasurer; Contact for Sustaining Members) [since 11.2015]
-* Anita Graser (Design Advisor) [since 9.2013]
-* Paolo Cavallini [Chair 4.2018 - 4.2020; Vice chair 1.2008 - 3.2018]
-* Jürgen Fischer (Release Manager) [since 9.2013]
+* Marco Bernasocchi (Chair) [Chair since May 2020; Vice chair April 2018 - April 2020]
+* Alessandro Pasotti (Vice chair) [Vice chair since May 2020]
```
Comment (DelazJ) [inline source/site/getinvolved/governance/governance.rst]: This does also change

File: source/site/getinvolved/governance/governance.rst [hunk 2]
```diff
@@ -26,12 +26,12 @@ To contact the board, please write to board@qgis.org.
...
+* Andreas Neumann (Treasurer; Contact for Sustaining Members) [since Nov. 2015]
+* Anita Graser (Design Advisor) [since Sept. 2013]
```
Comment (DelazJ) [inline source/site/getinvolved/governance/governance.rst]: As well as here

File: source/site/getinvolved/governance/governance.rst (hunk 1 above)
Comment (haubourg) [reply source/site/getinvolved/governance/governance.rst]: Sorry, what do you mean here Harrissou ? Do I have to change something ?

Comment (DelazJ) [reply source/site/getinvolved/governance/governance.rst]: According to last election, Alessandro is no longer vice-chair. Anita is. so end/start dates (and role) for both should be updated.

**Comment by haubourg** [comment]
Should be good now.

File: source/site/getinvolved/governance/governance.rst [hunk 3]
```diff
@@ -26,12 +26,12 @@ To contact the board, please write to board@qgis.org.
...
 --------------------------
 
-* Marco Bernasocchi (Chair) [Chair since 5.2020; Vice chair 4.2018 - 4.2020]
-* Alessandro Pasotti (Vice chair) [Vice chair since 5.2020]
-* Andreas Neumann (Treasurer; Contact for Sustaining Members) [since 11.2015]
-* Anita Graser (Design Advisor) [since 9.2013]
-* Paolo Cavallini [Chair 4.2018 - 4.2020; Vice chair 1.2008 - 3.2018]
-* Jürgen Fischer (Release Manager) [since 9.2013]
+* Marco Bernasocchi (Chair) [Chair since May 2020; Vice chair April 2018 - April 2020]
+* Anita Graser (Vice chair) [Design Advisor Sept. 2013 - Sept. 2022]**
```
Comment (agiudiceandrea) [inline source/site/getinvolved/governance/governance.rst:30]: `Anita Graser (Vice chair) [Design Advisor Sept. 2013 - Sept. 2022]**`

@haubourg, what's the meaning of the two trailing asterisks?

Comment (haubourg) [reply source/site/getinvolved/governance/governance.rst:30]: It might be a typo

Comment (DelazJ) [reply source/site/getinvolved/governance/governance.rst:30]: Fixed in ee02ba5

**Pull request merged**